src/App.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from 'react';
+import {
+  BrowserRouter,
+  Routes,
+  Route
+} from "react-router-dom";
+import Layout from './containers/Layout';
+import Home from './pages/Home';
+import Videos from './pages/Videos';
+import { VideoContext } from './resources/state/videoContext';
+
+const YOUTUBE_API_URL= 'https://www.googleapis.com/youtube/v3/search';
+const KEY = 'AIzaSyDcmvhajFZY_7MjXmJJVAkHxBXy1gsR3Ps';
```

Comment:
I would recommend placing these keys somewhere else. You could look into adding these variables to ENV variables. (suggestion: Github pages has the configuration of variables inside your repo)